### PR TITLE
fix: OR-1663 add response examples and documentation

### PR DESCRIPTION
**Pull request by janlesage**
Refactor into naming conventions
Added swagger examples and endpoint annotations

### CODE REVIEW

File: src/AssociationRegistry.Invitations.Api/Uitnodigingen/Controllers/GetUitnodigingVoorPersoon.cs [hunk 1]
```diff
@@ -48,3 +60,28 @@ public GetUitnodigingVoorPersoon(IQuerySession session)
...
+
+internal class UitnodigingsDetailExamples : IExamplesProvider<UitnodigingsDetail>
+{
+    public UitnodigingsDetail GetExamples()
+        => new()
+        {
+            UitnodigingId = Guid.NewGuid(),
+            VCode = "V0000001",
+            Boodschap = "<boodschap voor uitgenodigde>",
+            Status = "<status>",
```
Comment (koenmetsu) [inline src/AssociationRegistry.Invitations.Api/Uitnodigingen/Controllers/GetUitnodigingVoorPersoon.cs:72]: Add random status?

**Review by talboomerik — APPROVED** [review]

File: src/AssociationRegistry.Invitations.Api/Uitnodigingen/Controllers/GetUitnodigingenVoorVereniging.cs [hunk 2]
```diff
@@ -27,8 +28,14 @@ public GetUitnodigingenVoorVereniging(IQuerySession session)
...
+    /// <response code="200">Bevat een lijst met uitnodigingen voor de gevraagde vereniging.</response>
+    /// <response code="500">Er is een interne fout opgetreden.</response>
     /// <returns></returns>
     [HttpGet("verenigingen/{vcode}/uitnodigingen")]
+    [SwaggerResponseExample(StatusCodes.Status200OK, typeof(UitnodigingenResponseExamples))]
+    [SwaggerResponseExample(StatusCodes.Status500InternalServerError, typeof(InternalServerErrorResponseExamples))]
+    [ProducesResponseType(typeof(UitnodigingenResponse), StatusCodes.Status200OK)]
+    [ProducesResponseType(typeof(ProblemDetails), StatusCodes.Status500InternalServerError)]
     [ProducesJson]
     [ConsumesJson]
```
Comment (koenmetsu) [inline src/AssociationRegistry.Invitations.Api/Uitnodigingen/Controllers/GetUitnodigingenVoorVereniging.cs:40]: does not consume json?

File: src/AssociationRegistry.Invitations.Api/Uitnodigingen/Controllers/GetUitnodigingenVoorVereniging.cs [hunk 3]
```diff
@@ -44,3 +51,54 @@ public async Task<IActionResult> Get([FromRoute] string vCode, CancellationToken
...
+                    },
+                    Status = "<status>",
+                    DatumRegistratie = DateTime.Today.AddDays(-1).ToLongDateString(),
+                    DatumLaatsteAanpassing = DateTime.Today.ToLongDateString()
+                },
+                new Responses.Uitnodiging()
+                {
+                    UitnodigingId = Guid.NewGuid(),
+                    VCode = "V0000001",
+                    Boodschap = "<boodschap voor uitgenodigde>",
```
Comment (koenmetsu) [inline src/AssociationRegistry.Invitations.Api/Uitnodigingen/Controllers/GetUitnodigingenVoorVereniging.cs:86]: Add some boodschap

File: src/AssociationRegistry.Invitations.Api/Uitnodigingen/Controllers/RegistreerUitnodiging.cs [hunk 4]
```diff
@@ -28,22 +28,22 @@ public RegistreerUitnodiging(IDocumentStore store, IClock clock)
...
     /// <response code="500">Er is een interne fout opgetreden.</response>
     /// <returns></returns>
+    [HttpPost("uitnodigingen")]
     [SwaggerResponseExample(StatusCodes.Status201Created, typeof(RegistratieResponseExamples))]
     [SwaggerResponseExample(StatusCodes.Status400BadRequest, typeof(BadRequestProblemDetailsExamples))]
     [SwaggerResponseExample(StatusCodes.Status500InternalServerError, typeof(InternalServerErrorResponseExamples))]
     [ProducesResponseType(typeof(RegistratieResponse), StatusCodes.Status201Created)]
     [ProducesResponseType(typeof(ValidationProblemDetails), StatusCodes.Status400BadRequest)]
     [ProducesResponseType(typeof(ProblemDetails), StatusCodes.Status500InternalServerError)]
     [ProducesJson]
```
Comment (koenmetsu) [inline src/AssociationRegistry.Invitations.Api/Uitnodigingen/Controllers/RegistreerUitnodiging.cs:46]: ConsumesJson

**Review by koenmetsu — CHANGES_REQUESTED** [review]

File: src/AssociationRegistry.Invitations.Api/Uitnodigingen/Controllers/GetUitnodigingVoorPersoon.cs (hunk 1 above)
Comment (janlesage) [reply src/AssociationRegistry.Invitations.Api/Uitnodigingen/Controllers/GetUitnodigingVoorPersoon.cs:72]: OK

**Review by koenmetsu — APPROVED** [review]

**Pull request merged**